src/training.py

```diff
@@ -269,21 +270,62 @@ def run_experiment(
         - test_pred_dist: The test prediction distribution
         - mcc: The test Matthews correlation coefficient
     """
+
+    trains = []
+    target_train = None
+    target_valid = None
+    target_test = None
     th.manual_seed(1984)
 
-    train_data, valid_data, test_data = create_datasets(
-        symbol,
-        seq_len=seq_len,
-        fixed_scaling=[(7, 3000.), (8, 12.), (9, 31.)],
-        log_splits=log_splits,
-        root=f"{root}/data/clean"
+    for symbol in train_symbols:
+        train_data, valid_data, test_data = create_datasets(
+            symbol,
+            seq_len=seq_len,
+            fixed_scaling=[(7, 3000.), (8, 12.), (9, 31.)],
+            log_splits=log_splits,
+            root=f"{root}/data/clean"
+        )
+
+        trains.append(train_data)
+        if symbol == target_symbol:
+            target_train = train_data
+            target_valid = valid_data
+            target_test = test_data
+
+    concatted_trains = ConcatDataset(trains)
+
+    train_loader = DataLoader(concatted_trains, batch_size=batch_size, shuffle=True)
+    valid_loader = DataLoader(target_valid, batch_size=batch_size, shuffle=False)
+    test_loader = DataLoader(target_test, batch_size=batch_size, shuffle=False)
```

Comment:
Consider moving the data logic (lines 280-299) to their own function (app has a trainer_tools folder, can be added to utils or its own file). A) makes reading the trainer easier and B) means we can use things piecemeal while getting things working correctly before making them part of the overall training cycle.

Reply:
this is a larger refactor - let's get to it if we have time but suggest keeping as is for now. The code is going to be thrown away in ~1 week anyway

Reply:
Nothing is getting thrown away :P 

That being said, that is fine, I may pull it out while I wrap up my work...